frontrunner_sdk/models/order.py

```diff
@@ -1,11 +1,45 @@
-from dataclasses import dataclass
-from typing import Literal
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Literal, Sequence
+
+from pyinjective.proto.exchange.injective_derivative_exchange_rpc_pb2 import DerivativeOrderHistory
 
 OrderType = Literal["buy", "sell", "stop_buy", "stop_sell", "take_buy", "take_sell", "buy_po", "sell_po"]
 OrderState = Literal["booked", "partial_filled", "filled", "canceled"]
 OrderExecutionType = Literal["limit", "market"]
 
 
+class FrOrderType(Enum):
```

Comment:
I'd say let's take over the existing `OrderType` and define it as...

```python
OrderType = Literal["buy_long", "sell_long", "buy_short", "sell_short"]
```

And for the existing stuff, we make that

```python
InjectiveOrderType = Literal["buy", "sell", "stop_buy", "stop_sell", "take_buy", "take_sell", "buy_po", "sell_po"]
```

Our sdk code should take "priority" in naming, since that's the primary point of interaction. The "Injective" order type is just for type safety, and devs should not have to deal with it directly.